### PR TITLE
AdminUI: updates to Custom Group & Field

### PR DESCRIPTION
**Pull request by aydun**
Overview
----------------------------------------
This enhances the Administer Custom Group and Field pages in the AdminUI extension.

Before
----------------------------------------
It is not always obvious which custom group contains a specific custom field resulting in multiple rounds of opening up the field listing for a group, realising it is the wrong one, going back and trying again.

After
----------------------------------------
Custom field names are listed on the custom group page along with a filter making it very quick to find and edit a field.

Also has a few tweaks/standardisations in line with other AdminUI pages.

Technical Details
----------------------------------------
The field listing is formatted by replacing commas with '&lt;br&gt;'.  If a field label contains a comma it will have wrong line breaks, but that is fairly unlikely and not a big deal if it happens.  Ideally, the fields would be available as an array to allow formatting, but until then, this is good enough. This does not affect filtering.

Comments
----------------------------------------

### CODE REVIEW

File: ext/civicrm_admin_ui/managed/SavedSearch_Administer_Custom_Groups.mgd.php [hunk 1]
```diff
@@ -133,13 +146,15 @@
...
+                  'path' => '',
+                  'condition' => [],
                 ],
               ],
               'type' => 'buttons',
               'alignment' => 'text-right',
             ],
             [
-              'text' => '',
+              'text' => E::ts(''),
```
Comment (colemanw) [inline ext/civicrm_admin_ui/managed/SavedSearch_Administer_Custom_Groups.mgd.php:157]: Is this a bug in the Export API?

File: ext/civicrm_admin_ui/managed/SavedSearch_Administer_Custom_Groups.mgd.php [hunk 2]
```diff
@@ -176,9 +195,13 @@
               FALSE,
             ],
           ],
+          'placeholder' => 5,
         ],
         'acl_bypass' => FALSE,
       ],
+      'match' => [
+        'name',
```
[the remark this answers is not in the document]
Comment (colemanw) [reply ext/civicrm_admin_ui/managed/SavedSearch_Administer_Custom_Groups.mgd.php:203]: I'm pretty sure the latest `master` will generate it like this

**Comment by aydun** [comment]
Thanks - updated.

**Comment by colemanw** [comment]
@aydun I just went down a wee rabbit hole with this one. Because the raw data (which has the results of `GROUP_CONCAT(fields)` already as an array) [*is* available in the rewrite function](https://github.com/civicrm/civicrm-core/blob/46ae5f29d06733c090d7e1f00543ea89428d672a/ext/search_kit/Civi/Api4/Action/SearchDisplay/AbstractRunAction.php#L282)... yet just out of reach.
It *can* be assigned for use in the Smarty syntax, as it was in [this earlier draft of the PR](https://github.com/civicrm/civicrm-core/pull/22346), however our [security policy](https://github.com/civicrm/civicrm-core/pull/1371) prevents anything useful being done with it as php-based modifiers like `|implode` are disallowed.
The [list of modifiers we allow in strict mode](https://github.com/civicrm/civicrm-packages/blob/6672e68509b80554c1a27c1e843d4d69df139ef1/Smarty/Smarty.class.php#L229-L241) is much smaller than necessary (there's nothing insecure about `strtolower` or `uc_words` or `implode`). But I'm slightly hesitant to expand that list because it increases the size of our code contract (and if we ever switched from Smarty2 to some other template system, we might regret all the weird & wonderful search display rewrites we'd have to backward-support).
So that leaves me with... this PR as-is.

**Pull request merged**